ai21/errors.py

```diff
@@ -96,9 +96,9 @@ def __init__(self, key: str):
 
 class StreamingDecodeError(AI21Error):
     def __init__(self, chunk: str, error_message: Optional[str] = None):
-        message = f"Failed to decode chunk: {chunk} in stream. Please check the stream format"
+        message = f"Failed to decode chunk: {chunk} in stream. Please check the stream format. "
```

Comment:
Its weird to add a space at the end of the string "just in case"
You can do something like - 

```
if error_message:
    message = f"{message} Error: {error_message}"
```